src/deploy/standalone/logrotate_noobaa.conf

```diff
@@ -40,7 +38,6 @@
     create 644 root root
     sharedscripts
     postrotate
-        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-        /bin/kill -HUP `cat /var/run/rsyslogd.pid 2> /dev/null` 2> /dev/null || true
+        kill -HUP $(pidof rsyslogd) >/dev/null 2>&1 || true
     endscript
 }
```

Comment:
@naveenpaul1 Its not your change but I think you should modify the "size" and "daily" options also.

Remove "daily" and keep "size" option only. As per man page of logrotate , which I have also tested, "daily|hourly|monthly" will not matter if you have provided "size" option. Specially in your case, you should anyway only use rotation based on "size" only.

Reply:
@aspandey I think rotating daily basis will organise the logs more readable format.

Reply:
I think decision taken based on  "size" would be better. What I wanted to say in previous comment is that, "size"  option will override "daily"  and only "size" will be applicable so why to mention "daily" at all.